docs/code-examples/transform3d_simple.py

```diff
@@ -7,14 +7,10 @@
 
 rr.init("rerun_example_transform3d", spawn=True)
 
-origin = [0, 0, 0]
-base_vector = [0, 1, 0]
+rr2.log("base", rr2.Arrows3D([0, 1, 0]))
```

Comment:
As I user I would find this code very confusing. An arrow defined as three values? Is this the tip or the base? Neither!

I think we should be more explicit in all our examples (including both the origin and the vector for all arrows), and also consider changing the API explicitly mention the vector i.e. `rr2.Arrows3D(vector=[0, 1, 0], origin=[0, 0, 0])`

Reply:
New issue: https://github.com/rerun-io/rerun/issues/3402
But I think this code can still be improved in this PR

Reply:
About to be fixed by Andreas' PR